akka-cluster-sharding-typed/src/main/scala/akka/cluster/sharding/typed/javadsl/ClusterSharding.scala

```diff
@@ -442,6 +442,23 @@ object EntityTypeKey {
 @DoNotInherit abstract class EntityRef[-M] extends RecipientRef[M] {
   scaladslSelf: scaladsl.EntityRef[M] with InternalRecipientRef[M] =>
 
+  /**
+   * The identifier for the particular entity referenced by this EntityRef.
+   */
+  def getEntityId: String = entityId
```

Comment:
I can't say I'm familiar with recent changes in Java idiom, so apologies if `get` is frowned upon.  `getDataCenter` does avoid a collision with the `scaladsl` and Impl `dataCenter`.

Reply:
No, that's how we usually do it.